src/macros.rs

```diff
@@ -52,7 +52,7 @@ macro_rules! include_spirv {
             $crate::ShaderModuleDescriptor {
                 label: Some($($token)*),
                 source: $crate::util::make_spirv(include_bytes!($($token)*)),
-                experimental_translation: false,
+                flags: $crate::ShaderFlags::VALIDATION,
```

Comment:
Does validation still fall through if it can't parse stuff? If not I'd rather this be user controllable.

Reply:
It still falls, yes. But at the same time - the flag is fully user-controllable. The macro is just a helper, it's not the actual wgpu-rs API.

Reply:
Yeah I know, it just seems like if there is a macro available, users use it, so it's worthwhile keeping it in mind. But it failing out is fine.